src/Components/ProductOverview/ProductOverview.jsx

```diff
@@ -1,44 +1,30 @@
-import React, { Component } from 'react';
+/* eslint-disable import/no-named-as-default */
+import React from 'react';
 import { connect } from 'react-redux';
 import ProductInfo from './ProductInfo/ProductInfo';
 import DescriptionFeaturesShare from './DescriptionFeaturesShare/DescriptionFeaturesShare';
-import './ProductOverview.css';
+import ImageGallery from './ImageGallery/ImageGallery';
+import './ProductOverview.scss';
 
-class ProductOverview extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-    };
-  }
-
-  render() {
-    return (
-      <div className="product-overview">
-        <div className="container-fluid">
-          <div className="row">
-            <div className="border col-12 col-sm-8 col-lg-8 col-xl-8 image-gallery">
-              ImageGallery
-            </div>
-            <div className="border col-12 col-sm-4 col-lg-4 col-xl-4 product-info">
-              <ProductInfo />
-            </div>
-          </div>
-          <div className="row border">
-            <DescriptionFeaturesShare />
-          </div>
+const ProductOverview = () => (
+  <div className="product-overview">
+    <div className="container-fluid">
+      <div className="row">
+        <div className="border col-12 col-sm-8 col-lg-8 col-xl-8 image-gallery">
+          <ImageGallery />
+        </div>
+        <div className="border col-12 col-sm-4 col-lg-4 col-xl-4 product-info">
+          <ProductInfo />
         </div>
       </div>
-    );
-  }
-}
+      <div className="row border">
+        <DescriptionFeaturesShare />
+      </div>
+    </div>
+  </div>
+);
 
 const mapStateToProps = (state) => ({
```

Comment:
I'm pretty sure that you can get rid of this, I did so in my section/area/component and everything still works fine

Reply:
Yeah you can. I was leaving it just in case I might needed for something later one, but that seems unlikely. thanks for letting me know!